data/json/requirements/materials.json

```
@@ -151,10 +151,9 @@
       [
         [ "feather", 10 ],
         [ "duct_tape", 40 ],
-        [ "paper", 40 ],
+        [ "cardboard", 10 ],
```

```suggestion
        [ "cardboard", 2 ],
```
A single paper cup produces 2 cardboard and should have enough for an arrow or two. A small cardboard box should be able to fletch several arrows, and a cardboard box large enough to hide in should make tons of them.

---

That recipe produces 10 arrows.

---

Ah, okay. It can probably stay at the current requirement then.